sections/shared/Layout/HomeLayout/Header.tsx

```diff
@@ -120,28 +120,43 @@ const Header: FC = () => {
 					</Links>
 					<MenuContainer>
 						<Link href={ROUTES.Markets.Home}>
-							<Button variant="primary" isRounded={false} size="sm">
+							<Button noOutline={true} size="sm">
```

Comment:
We can save a few keystrokes like this. Looks like there are other instances in this file where we could also do this.
```suggestion
							<Button noOutline size="sm">
```

Reply:
Thanks for the suggestions. I will clean all of them.